### PR TITLE
virttest/test_setup: make sure all pids killed

### PR DESCRIPTION
**Pull request by PandaWei**
Sleep 3 sec is not enough sometimes,
It's necessary to ensure all of pids end up before follow-up steps.

### CODE REVIEW

File: virttest/test_setup.py [hunk 1]
```diff
@@ -1675,7 +1675,16 @@ def cleanup(self):
...
-            # give time to wait port released by egd.pl
-            time.sleep(3)
+
+            def _all_killed():
+                for pid in self.env.data["egd_pids"]:
+                    if utils.pid_is_alive(pid):
+                        return False
+                return True
+            # wait port released by egd.pl
+            if utils.wait_for(_all_killed, timeout=60) is not True:
```
Comment (Hao-Liu) [inline virttest/test_setup.py]: Why bother wait another 3 seconds if you've already waited 1 minute? I think just `utils.wait_for(_all_killed, timeout=60)` should be okay.

Comment (PandaWei) [inline virttest/test_setup.py]: Thanks for your response and comment.
updated it.

**Comment by Hao-Liu** [comment]
LGTM

**Comment by will-Do** [comment]
Good

**Pull request merged**